### DISCUSSION TITLE
LC and MS methods

### DISCUSSION THREAD

**Discussion by KeepCalmAndRushB** [MannLabs/alphadia#153] [Ideas]
For each run, please generate also a table with the LC and MS method used

**Comment by mschwoer** [comment]
we see this as a feature in AlphaRaw rather than AlphaDIA .. 
tracked internally here https://github.com/orgs/MannLabs/projects/12/views/1?filterQuery=-last-updated%3A14days&pane=issue&itemId=111356909
